src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+  const stylesArray = sourceString.split(';');
+  // розрізали на ("властивість: значення")
+
+  stylesArray.forEach((style) => {
+    const [property, ...valueParts] = style.split(':');
+    // розбиває кожен елемент масиву на дві частини - властивість ';' значення
+    const trimmedProperty = property.trim();
+    const trimmedValue = valueParts.join(':').trim();
```

Comment:
[CODE PRACTICE] - Since the task does not require handling multiple colons within the property values, you can simplify the code by directly assigning 'valueParts[0]' to 'trimmedValue' after trimming it, instead of joining the array back into a string.